Create david_stafford_10xgenomics.md

dlas opened the pull request:
verification link: https://www.linkedin.com/in/dstaff

natdempk commented:
Hi @dlas , thank you for your pull request.  To add this signature to the pledge, we need to verify that this signature accurately represents the owner of this GitHub account.

Could you please put a link to your GitHub account on an established webpage (such as your Keybase or LinkedIn profile, or personal site) that represents you by the name and affiliation in your signature? Please reply with the URL of that webpage, and we'll take a look.

Thanks!

dlas commented:
I updated my linkedin profile, https://www.linkedin.com/in/dstaff to
include a pointer to my github account in the summary section.  I also
added  reverse pointer from the github account (https://github.com/dlas) to
that linked account.

Thanks.


On Wed, Dec 21, 2016 at 12:33 PM, Nathaniel Dempkowski <
notifications@github.com> wrote:

> Hi @dlas <https://github.com/dlas> , thank you for your pull request. To
> add this signature to the pledge, we need to verify that this signature
> accurately represents the owner of this GitHub account.
>
> Could you please put a link to your GitHub account on an established
> webpage (such as your Keybase or LinkedIn profile, or personal site) that
> represents you by the name and affiliation in your signature? Please reply
> with the URL of that webpage, and we'll take a look.
>
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/neveragaindottech/neveragaindottech.github.io/pull/2561#issuecomment-268631862>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACsrPdPG4uPcHun0idfoktmxzQIk0oDCks5rKY0EgaJpZM4LTQ9g>
> .
>

natdempk commented:
Cheers!
